### ISSUE TITLE
Installation Error

### ISSUE THREAD

**Issue by captain1fs** [SkoltechAI/fewbit#2]
When I installed fewbit, I got the following error：

pip install -U fewbit
ERROR: Could not find a version that satisfies the requirement fewbit (from versions: none)
ERROR: No matching distribution found for fewbit

pip install -U https://github.com/SkoltechAI/fewbit.git
Collecting https://github.com/SkoltechAI/fewbit.git
  Downloading https://github.com/SkoltechAI/fewbit.git
     \ 223.1 kB 2.1 MB/s 0:00:00
  ERROR: Cannot unpack file /tmp/pip-unpack-a3ukc7tp/fewbit.git (downloaded from /tmp/pip-req-build-aipk8zcb, content-type: text/html; charset=utf-8); cannot detect archive format
ERROR: Cannot determine archive format of /tmp/pip-req-build-aipk8zcb

So how should I do?Thanks.

**Comment by daskol** [comment]
Thanks! This is a typo. Actual command follows.
```python
pip install -U git+https://github.com/SkoltechAI/fewbit.git
```
Also, I'd recommend to switch to `wheels` branch (see #3) where we are trying to build manylinux wheels for different CUDAs and PyTorch versions.

**Comment by captain1fs** [comment]
Thanks for you reply! After some tries, I installed fewbit Successfully. But when I try to use GELU to replace ReLU in some models for training, I got errors as follows:

NotImplementedError: Could not run 'fewbit::gelu' with arguments from the 'CUDA' backend. This could be because the operator doesn't exist for this backend, or was omitted during the selective/custom build process (if using custom build). If you are a Facebook employee using PyTorch on mobile, please visit https://fburl.com/ptmfixes for possible resolutions. 'fewbit::gelu' is only available for these backends: [Dense, UNKNOWN_TENSOR_TYPE_ID, QuantizedXPU, UNKNOWN_TENSOR_TYPE_ID, UNKNOWN_TENSOR_TYPE_ID, UNKNOWN_TENSOR_TYPE_ID, UNKNOWN_TENSOR_TYPE_ID, UNKNOWN_TENSOR_TYPE_ID, SparseCPU, SparseCUDA, SparseHIP, UNKNOWN_TENSOR_TYPE_ID, UNKNOWN_TENSOR_TYPE_ID, UNKNOWN_TENSOR_TYPE_ID, SparseVE, UNKNOWN_TENSOR_TYPE_ID, NestedTensorCUDA, UNKNOWN_TENSOR_TYPE_ID, UNKNOWN_TENSOR_TYPE_ID, UNKNOWN_TENSOR_TYPE_ID, UNKNOWN_TENSOR_TYPE_ID, UNKNOWN_TENSOR_TYPE_ID, UNKNOWN_TENSOR_TYPE_ID].

CPU: registered at /xfs/home/tensor_zy/zhangjie/Memopt/fewbit/fewbit/cpu/gelu.cc:74 [kernel]
BackendSelect: fallthrough registered at ../aten/src/ATen/core/BackendSelectFallbackKernel.cpp:3 [backend fallback]
Python: registered at ../aten/src/ATen/core/PythonFallbackKernel.cpp:133 [backend fallback]
Named: registered at ../aten/src/ATen/core/NamedRegistrations.cpp:7 [backend fallback]
Conjugate: registered at ../aten/src/ATen/ConjugateFallback.cpp:18 [backend fallback]
Negative: registered at ../aten/src/ATen/native/NegateFallback.cpp:18 [backend fallback]
ZeroTensor: registered at ../aten/src/ATen/ZeroTensorFallback.cpp:86 [backend fallback]
ADInplaceOrView: fallthrough registered at ../aten/src/ATen/core/VariableFallbackKernel.cpp:64 [backend fallback]
AutogradOther: fallthrough registered at ../aten/src/ATen/core/VariableFallbackKernel.cpp:35 [backend fallback]
AutogradCPU: fallthrough registered at ../aten/src/ATen/core/VariableFallbackKernel.cpp:39 [backend fallback]
AutogradCUDA: fallthrough registered at ../aten/src/ATen/core/VariableFallbackKernel.cpp:47 [backend fallback]
AutogradXLA: fallthrough registered at ../aten/src/ATen/core/VariableFallbackKernel.cpp:51 [backend fallback]
AutogradMPS: fallthrough registered at ../aten/src/ATen/core/VariableFallbackKernel.cpp:59 [backend fallback]
AutogradXPU: fallthrough registered at ../aten/src/ATen/core/VariableFallbackKernel.cpp:43 [backend fallback]
AutogradHPU: fallthrough registered at ../aten/src/ATen/core/VariableFallbackKernel.cpp:68 [backend fallback]
AutogradLazy: fallthrough registered at ../aten/src/ATen/core/VariableFallbackKernel.cpp:55 [backend fallback]
Tracer: registered at ../torch/csrc/autograd/TraceTypeManual.cpp:295 [backend fallback]
AutocastCPU: fallthrough registered at ../aten/src/ATen/autocast_mode.cpp:481 [backend fallback]
Autocast: fallthrough registered at ../aten/src/ATen/autocast_mode.cpp:324 [backend fallback]
Batched: registered at ../aten/src/ATen/BatchingRegistrations.cpp:1064 [backend fallback]
VmapMode: fallthrough registered at ../aten/src/ATen/VmapModeRegistrations.cpp:33 [backend fallback]
Functionalize: registered at ../aten/src/ATen/FunctionalizeFallbackKernel.cpp:89 [backend fallback]
PythonTLSSnapshot: registered at ../aten/src/ATen/core/PythonFallbackKernel.cpp:137 [backend fallback]

Training using CPU is right. Does fewbit not support CUDA? Thanks.

**Comment by captain1fs** [comment]
Thanks for your reply! I have installed fewbit successfully.But I got errors when using Piece-wise Activation Functions, as follows:

- RuntimeError: fewbit::relu() expected at most 1 argument(s) but received 2 argument(s). Declaration: fewbit::relu(Tensor(a!) self) -> (Tensor(a!))
- RuntimeError: fewbit::hardsigmoid() expected at most 1 argument(s) but received 2 argument(s). Declaration: fewbit::hardsigmoid(Tensor(a!) self) -> (Tensor(a!))

It is right when using Continous Activation Functions. So what should I do? Thanks!
